FreeRTOS/FreeRTOS
# Update demo to latest +TCP dev/IPv6_integration

Description
-----------
This PR updates the +TCP demos to be compatible with the latest changes related to picking latest cmake changes from +TCP mainline branch to +TCP dev/IPv6_integration.

Test Steps
-----------
Build and run the demos on Jenkins.

Checklist:
----------




By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.
